front/lib/models/workspace.ts

```diff
@@ -67,6 +68,10 @@ Workspace.init(
     plan: {
       type: DataTypes.STRING,
     },
+    stripeCustomerId: {
```

Comment:
If it's possible to move tihs to the Subscription object instead of Workspace I think it would be a massive win (avoid leaking Stripe abstractions on our central Workspace(Type) logic)?